packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -534,6 +534,46 @@ void main() {
         equals('TextInputAction.done'));
   });
 
+  testWidgets('connection is closed when TextInputClient.onConnectionClosed message received', (WidgetTester tester) async {
+    Future<void> verifyVisibility(bool expectedVisibility) async {
+      expect(tester.testTextInput.isVisible, expectedVisibility);
+    }
+
+    await tester.pumpWidget(
+      MediaQuery(
+        data: const MediaQueryData(devicePixelRatio: 1.0),
+        child: Directionality(
+          textDirection: TextDirection.ltr,
+          child: FocusScope(
+            node: focusScopeNode,
+            autofocus: true,
+            child: EditableText(
+              backgroundCursorColor: Colors.grey,
+              controller: controller,
+              focusNode: focusNode,
+              maxLines: 1, // Sets text keyboard implicitly.
```

Comment:
Why this?

Reply:
I want to it to be as close to the previous test as possible. I usually find it easier to troubleshoot when there is an error, if some other tests also have the same setup.

I can change it to another widget if you want to :) 

One thing I'll add is also: "switching between two editable text fields after connection closed". I believe it might be good to test that case.